test/uri_comparison_test.cpp

```diff
@@ -70,7 +70,7 @@ TEST(uri_comparison_test, inequality_test) {
 TEST(uri_comparison_test, less_than_test) {
   // lhs is lexicographically less than rhs
   network::uri lhs("http://www.example.com/");
-  network::uri rhs("http://www.example.org/");
+  network::uri rhs("http://www.example.com/path");
```

Comment:
Was there an issue with the original test? I'd prefer if you added this as a new test case instead of modifying an existing one.

Reply:
Yeah sure I should have added that case instead of replacing the existing one. 

I did a mistake when re-implementing `operator<` before deciding to use the `string_view`s operators. I think now that this is using `string_view::operator<` there is no need to increase test coverage here.

Reply:
Reverted in 1a55ecf